Dockerfile

```diff
@@ -18,9 +18,9 @@ FROM centos:7
 RUN yum install -y java-1.8.0-openjdk-devel
 ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
 
-ARG TEST_JAR_VERSION
-ENV TEST_JAR_VERSION 2.0.0-SNAPSHOT
-ENV TEST_JAR_PATH /opt/accumulo-testing-${TEST_JAR_VERSION}-shaded.jar
+ENV HADOOP_API_JAR /opt/hadoop-client-api.jar
+ENV HADOOP_RUNTIME_JAR /opt/hadoop-client-runtime.jar
+ENV TEST_JAR_PATH /opt/accumulo-testing-shaded.jar
 ENV ACCUMULO_CLIENT_PROPS /opt/conf/accumulo-client.properties
 ENV TEST_PROPS /opt/conf/accumulo-testing.properties
 ENV TEST_LOG4J /opt/conf/log4j.properties.example
```

Comment:
Could copy `log4j.properties.example` and `log4j.properties` into docker file and have script fallback to example if regular file does not exists.   I think the docker `COPY` command can take multiple files.